package.json

```diff
@@ -37,6 +37,9 @@
     "ts-jest": "^26.4.0",
     "typescript": "^3.9.3"
   },
+  "engines": {
+    "node": ">= 12"
```

Comment:
FWIW, I don't think we can make 12 the minimum if we plan to adopt in ember-template-lint (it supports "all currently supported Node versions" which atm includes Node 10), I'd like this to be:

```js
    "node": "10.* || 12.* || >= 14"
```

Reply:
That seems fine by me. I was hoping you'd comment on this. Will change.

Reply:
Done.